### PR TITLE
fix(infra-monitoring-volumes): add missing inodes columns

### PR DESCRIPTION
**Pull request by H4ad**
## Pull Request

---

Adding missing Inodes columns, I also removed the prefix of `Volume` to be able to have all columns in the UI without any horizontal scroll for 1920px.

Before / After

image

Fixes https://github.com/SigNoz/engineering-pod/issues/5333

---

### ✅ Change Type
_Select all that apply_

- [ ] ✨ Feature
- [x] 🐛 Bug fix
- [ ] ♻️ Refactor
- [ ] 🛠️ Infra / Tooling
- [ ] 🧪 Test-only

---

None

It was created without the Inodes columns.

Just add the columns.

---

- Tests added/updated: No
- Manual verification: Yes
- Edge cases covered: -

---

- Blast radius: Infrastructure Monitoring - Volumes
- Potential regressions: Unknown
- Rollback plan: Revert this commit

---

| Field | Value |
|------|-------|
| Deployment Type | Cloud / OSS / Enterprise |
| Change Type | Bug Fix |
| Description | On Infrastructure Monitoring, the columns for INodes was missing for the Volumes Category. |

---

### 📋 Checklist
- [ ] Tests added or explicitly not required
- [x] Manually tested
- [ ] Breaking changes documented
- [ ] Backward compatibility considered

### CODE REVIEW

**Review by AshwinBhatkal — DISMISSED** [review]

File: frontend/src/container/InfraMonitoringK8s/Volumes/table.config.tsx [hunk 1]
```diff
@@ -105,9 +105,9 @@ export const k8sVolumesColumnsConfig: TableColumnDef<K8sVolumesData>[] = [
 	},
 	{
 		id: 'usage',
-		header: 'Volume Utilization',
+		header: 'Utilization',
```
Comment (nikhilmantri0902) [inline frontend/src/container/InfraMonitoringK8s/Volumes/table.config.tsx]: Utilization should be named Used instead, Utilization refers to percentage but we report this value in bytes in this column.

**Review by nikhilmantri0902 — APPROVED** [review]
LGTM

**Review by AshwinBhatkal — APPROVED** [review]
